### PR TITLE
Fix breaking changes on 7.3.0-post

### PR DESCRIPTION
**Pull request by utkarsh5474**
# Description

To be merged as a single commit.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

https://jenkins.confluent.io/job/cp-ansible-on-demand/713/
https://jenkins.confluent.io/job/cp-ansible-on-demand/712/

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Comment by erikgb** [comment]
@utkarsh5474 I just ran a successful check mode pipeline using the 7.3.x branch. 👍 Will I be able to do the same using the 7.3.0-post branch after this PR merged?

File: docs/VARIABLES.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ Default:  /tmp
 
 ***
 
+***
+
+### fetch_logs_path
```
Comment (wadhwa1) [inline docs/VARIABLES.md:25]: I think, we have double entry of fetch_logs_path

**Review by wadhwa1 — APPROVED** [review]
one minor comment on duplicate variable entry, rest LGTM 👍 
thanks

**Review by nsharma-git — APPROVED** [review]

**Comment by utkarsh5474** [comment]
Yeah @erikgb, that's the goal.

**Comment by erikgb** [comment]
Works with 7.3.0-post also now. Thanks a lot, team! ❤️

**Pull request merged**